src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -17,6 +18,31 @@ namespace Microsoft.Diagnostics.Tracing.Parsers
     using Microsoft.Diagnostics.Tracing.Parsers.Clr;
     using System.Collections.Generic;
 
+    // TODO, AndrewAu, naming
+    public class CommittedUsageTraceData
```

Comment:
@brianrob what are your thoughts on where to put these?

Reply:
@brianrob, any opinions here? We simply added these classes outside the ClrTraceEventParser.